client/src/Context/isAuthContext.js

```diff
@@ -1,5 +1,35 @@
-import { createContext } from 'react';
+import React, { useContext, useState, createContext, useEffect } from 'react';
+import { element } from 'prop-types';
+import { get } from 'axios';
 
 const IsAuthContext = createContext(false);
 
+export const AuthProvider = ({ children }) => {
+  const authState = useState();
+  const [, setIsAuth] = authState;
```

Comment:
```suggestion
  const [isAuth, setIsAuth] = useState();
```

Reply:
i don't need the isAuth in this file so I just get the setIsAuth